.travis.yml

```diff
@@ -4,12 +4,14 @@ services:
   - docker
   
 before_install:
-  - docker build -t kpndigital/tox:py27_py35 .
```

Comment:
Definitely not keep, since that's misleading :) I removed the tag since all these changes are backwards compatible (only python3.6 and venv modules for 3.5 and 3.6 are added).

You prefer if people select `py27_py35_py36` explicitly?